src/py/run_generator.rs

```diff
@@ -208,31 +207,19 @@ pub fn run_generator(
     py: Python,
     program: &[u8],
     args: &[u8],
-    quote_kw: u8,
-    apply_kw: u8,
-    opcode_lookup_by_name: HashMap<String, Vec<u8>>,
```

Comment:
I don't think we should change this API without also changing the version number of `clvm_rs` (but I absolutely think we should change this API in this direction, I just think we should be careful to stay API compatbile within version numbers, even when building from git)

Reply:
and I have queued up other changes as well. For now, maybe we can leave these as unused parameters until we cut over